Plugins/Public/base_plugin/Main.cpp

```diff
@@ -2013,15 +2014,15 @@ void __stdcall CShip_destroy(CShip* ship)
 void BaseDestroyed(uint space_obj, uint client)
 {
 	returncode = DEFAULT_RETURNCODE;
-	map<uint, Module*>::iterator i = spaceobj_modules.find(space_obj);
+	auto& i = spaceobj_modules.find(space_obj);
 	if (i != spaceobj_modules.end())
 	{
 		returncode = SKIPPLUGINS;
 		i->second->SpaceObjDestroyed(space_obj);
 	}
 }
 
-void __stdcall HkCb_AddDmgEntry(DamageList *dmg, unsigned short p1, float& damage, enum DamageEntry::SubObjFate fate)
+void __stdcall HkCb_AddDmgEntry(DamageList *dmg, unsigned short sID, float& newHealth, enum DamageEntry::SubObjFate& fate)
```

Comment:
think you've missed the part where you change the references from `p1` to `sID` and from `damage` to `newHealth`

Reply:
Reverting the names back to normal.
Proper renaming of the parameters within the method happened in the #197